cves/kernel/CVE-2021-38202.yml

```diff
@@ -56,7 +56,17 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  This vulnerability is very basic, but has some dangerous implications: An 
+  unchecked, external string was being entered into method that calls 
+  "strlen()". In C, Strings must be null terminated (a null character must 
+  be at the end of every string, to signify its end). When strlen is called on
+  a string that is not null terminated, it will continue to look through memory 
+  until it hits a bit defined as null, which can result in undefined behavior. 
+  Since the input was from an external source, and was not verified, it was 
+  possible that it could not be properly null terminated, and thus could result
+  in the system behaving in an unintended fashion, and could even lead to a 
+  denial of service situation.
```

Comment:
Add reference to NFS/NFSD traffic and explain what it is and how the vulnerability impacts it

Reply:
Updated. Also fixed wording at the beginning of the description so that it is all in the same tense

Reply:
Change approved in external conversation.